src/payment/strategies/opy/opy-payment-strategy.ts

```diff
@@ -10,18 +10,35 @@ import StorefrontPaymentRequestSender from '../../storefront-payment-request-sen
 import PaymentStrategy from '../payment-strategy';
 
 import { isOpyPaymentMethod, ActionTypes } from './opy';
+import { OpyWidgetConfig } from './opy-library';
 import OpyError from './opy-payment-error';
+import OpyScriptLoader from './opy-script-loader';
 
 export default class OpyPaymentStrategy implements PaymentStrategy {
     constructor(
         private _store: CheckoutStore,
         private _orderActionCreator: OrderActionCreator,
         private _paymentMethodActionCreator: PaymentMethodActionCreator,
         private _storefrontPaymentRequestSender: StorefrontPaymentRequestSender,
-        private _paymentActionCreator: PaymentActionCreator
+        private _paymentActionCreator: PaymentActionCreator,
+        private _scriptLoader: OpyScriptLoader
     ) { }
 
-    initialize(_options?: PaymentInitializeOptions): Promise<InternalCheckoutSelectors> {
+    async initialize(options?: PaymentInitializeOptions): Promise<InternalCheckoutSelectors> {
+        if (options?.opy?.containerId) {
+            const { methodId, opy: { containerId } } = options;
+
+            const paymentMethod = this._store.getState().paymentMethods.getPaymentMethod(methodId);
```

Comment:
```suggestion
            const paymentMethod = this._store.getState().paymentMethods.getPaymentMethodOrThrow(methodId);
```

Reply:
I don't want it to throw an error, it will just not install the widget, which is not essential, and continue.